### DISCUSSION TITLE
p-tree Lazy Loading - Spinner hangs

### DISCUSSION THREAD

**Discussion by bigelowt** [primefaces/community#809] [PrimeNG] [answered]
Having trouble with loading property of p-tree.  Seems to hang until I click off of it.  Thought it was my shady coding, but I see the same behavior on PrimeNG Features page...
![image](https://github.com/primefaces/community/assets/2000651/bc255bd3-7316-46e4-b97c-d071585213b9)

Is there a setting I'm missing?

Cheers!

Troy

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi, 

The problem is related with the demo, component itself work as expected. We've just fixed the demo and the fix will be deployed today. 

https://github.com/primefaces/primeng/commit/efd161cfe67f3bbc838369b444686abf3cec94a6
